test/Feature/HLSLLib/firstbitlow.test

```diff
@@ -0,0 +1,66 @@
+#--- source.hlsl
+
+StructuredBuffer<uint4> In : register(t0);
+RWStructuredBuffer<uint4> Out : register(u1);
+
+[numthreads(1,1,1)]
+void main() {
+  Out[0] = firstbitlow(In[0]);
+  Out[1].xyz = firstbitlow(In[0].xyz);
```

Comment:
``` 
Out[1].xyz = firstbitlow(In[0].xyz);
Out[1].w = firstbitlow(In[0].w);
``` 
appear to be problematic on Intel gpus so the new suggestion is to combine those into a temporary and then write to Out[1]
```
uint4 Out1 = {firstbitlow(In[0].xyz), firstbitlow(In[0].w)};
Out[1] = Out1;
```